gpkit/solution_array.py

```diff
@@ -90,62 +97,51 @@ def sens(self, p):
             assert not subbed.exp
             return mag(subbed.c)
 
-    def table(self, tables=["cost", "freevariables", "sweepvariables",
+    def table(self, tables=["cost", "sweepvariables", "freevariables",
                             "constants", "sensitivities"], fixedcols=True,
-                            included_models=None, excluded_models=None):
+              latex=False, included_models=None, excluded_models=None):
         if isinstance(tables, Strings):
             tables = [tables]
         strs = []
-        if "cost" in tables:
-            strs += ["\nCost\n----"]
-            if len(self) > 1:
-                costs = ["%-8.3g" % c for c in self["cost"][:4]]
-                strs += [" [ %s %s ]" % ("  ".join(costs),
-                                         "..." if len(self) > 4 else "")]
-                cost_units = self.program[0].cost.units
+        for table in tables:
+            subdict = self.get(table, None)
+            table_title = self.table_titles[table]
+            if table == "cost":
+                strs += ["\n%s\n----" % table_title]
+                if len(self) > 1:
+                    costs = ["%-8.3g" % c for c in subdict[:4]]
+                    strs += [" [ %s %s ]" % ("  ".join(costs),
+                                             "..." if len(self) > 4 else "")]
+                    cost_units = self.program[0].cost.units
+                else:
+                    strs += [" %-.4g" % subdict]
+                    cost_units = self.program.cost.units
+                strs[-1] += unitstr(cost_units, into=" [%s] ", dimless="")
+                strs += [""]
+            elif not subdict:
```

Comment:
This means the `table` specified cannot be printed, right? Should we print a warning then?